README.md

```diff
@@ -1,68 +1,41 @@
-
-Members of the TSC and CommComm shall operate under Lazy Consensus as a collaborative unit in the Admin repo hereby referenced as the *Admin members*. Members shall establish appropriate guidelines for implementing Lazy Consensus (e.g. expected notification and review time periods) within the development process.
-
-For all votes, the vote must pass in both TSC and CommComm or else the motion fails. If an individual happens to be in both committees, then they would end up with two votes.
+The Admin repo is overseen by the [TSC](https://github.com/nodejs/TSC). Any action requiring consensus and voting will abide by the TSC process for that.
 
 ### Contacts for assistance
-- [@bnb](https://github.com/bnb) - **Tierney Cyren**, CommComm Chair
-- [@joesepi](https://github.com/joesepi) - **Joe Sepi**, CommComm OpenJS Representative
 - [@mhdawson](https://github.com/mhdawson) - **Michael Dawson**, TSC Chair
+- [@mcollina](https://github.com/mcollina) - **Matteo Collina**, TSC Vice Chair
+- [@joesepi](https://github.com/joesepi) - **Joe Sepi**, CommComm OpenJS Representative
 - [@mmarchini](https://github.com/mmarchini) - **Mary Marchini**, TSC OpenJS Representative
 
-### Admin members
-All active members of the TSC and CommComm.
-
-## Social Team
-
-The currently active members of the [Social Team](./social-team.md):
-
-- [@joesepi](https://github.com/joesepi) - **Joe Sepi**
-- [@Trott](https://github.com/Trott) - **Rich Trott**
-
 ## Node.js 1Password
 
-Thanks to 1Password's [open-source program](https://github.com/1Password/1password-teams-open-source), Node.js has been comped a paid version of 1Password.
-
-### Current account owners:
-There will always be two owners of this account - the TSC and CommComm chairpeople. These owners are expected to use their access only to maintain the account and not for access to credientials that they do not otherwise explicitly have access to through [Groups](#1password-groups) within.
-
-- [@bnb](https://github.com/bnb) - **Tierney Cyren**, CommComm Chair
-- [@mhdawson](https://github.com/mhdawson) - **Michael Dawson**, TSC Chair
+Thanks to 1Password's [open-source program](https://github.com/1Password/1password-teams-open-source), Node.js has been comped a paid version of 1Password, owned by the TSC.
 
 ### 1Password Groups
 
 Groups are created in 1Password with a 1:1 mapping of Node.js organization teams, as requested. Groups are asssigned to Vaults, which contain distinct sets of credentials that should only be used by the Groups that are provided access to them. If there is a team in the Node.js organization that would like to have a Group for your Node.js organization team, please PR an addition to the list below and request the current account owners as reviewers on the PR.
 
-#### Current Groups
-
-- **Social Team:** For the [Social Team](#social-team) to contain official account credientials.
-
 ## Node.js Project Calendar
```

Comment:
I would need help from @nodejs/tsc regarding who maintains the calendar now?

Reply:
Let's discuss that in a follow up PR.